src/libslic3r/GCode/PressureEqualizer.cpp

```diff
@@ -589,8 +590,7 @@ void PressureEqualizer::adjust_volumetric_rate(const size_t fist_line_idx, const
     if (line_idx == fist_line_idx || !m_gcode_lines[line_idx].extruding())
         // Nothing to do, the last move is not extruding.
         return;
-
-    std::array<float, size_t(GCodeExtrusionRole::Count)> feedrate_per_extrusion_role{};
+    std::array<float, size_t(ExtrusionRole::erCount)> feedrate_per_extrusion_role{};
```

Comment:
@@igiannakas
I have removed GCodeExtrusionRole as it's not synced with ExtrusionRole.
Hence the crash happens.
It's error prone in the future to have two version of ExtrusionRole so I have remove it.